wear/src/main/java/com/nexenio/sblecdemowear/WearDemoActivity.java

```diff
@@ -0,0 +1,51 @@
+package com.nexenio.sblecdemowear;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.wearable.activity.WearableActivity;
+
+import com.nexenio.sblecdemo.DemoLayout;
+
+import org.jetbrains.annotations.NotNull;
+
+import androidx.annotation.Nullable;
```

Comment:
Here again, androidx is not imported. Please don't mix androidx and legacy stuff

Reply:
Removed the legacy stuff :)